src/platform/packages/shared/kbn-data-grid-in-table-search/src/in_table_search_control.tsx

```diff
@@ -197,6 +192,7 @@ export const InTableSearchControl: React.FC<InTableSearchControlProps> = ({
         >
           <EuiButtonIcon
             data-test-subj={BUTTON_TEST_SUBJ}
+            buttonRef={(node: HTMLButtonElement) => (buttonRef.current = node)}
```

Comment:
```suggestion
            buttonRef={buttonRef}
```
Would it work to just pass `buttonRef` directly here?

Reply:
Yes, works too.